test/test_optim.py

```diff
@@ -200,29 +214,36 @@ def getPublicAttr(obj):
         self.assertEqual(getPublicAttr(optimizer), getPublicAttr(deepcopy(optimizer)))
 
     def _test_basic_cases(self, constructor, scheduler_constructors=None,
-                          ignore_multidevice=False):
+                          ignore_multidevice=False, constructor_accepts_maximize=False):
         if scheduler_constructors is None:
             scheduler_constructors = []
+        if constructor_accepts_maximize:
+            def wrapped_constructor(weight, bias):
+                return constructor(weight, bias, False)
```

Comment:
Is that ok for this one to always be False?

Reply:
I think it is fine, but no harm in testing more.